frontend/src/app/components/wp-table/configuration-modal/wp-table-configuration-relation-selector.ts

```diff
@@ -10,6 +10,7 @@ import {QueryFilterResource} from "core-app/modules/hal/resources/query-filter-r
 import {QueryOperatorResource} from "core-app/modules/hal/resources/query-operator-resource";
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/d971367a7a91bd5bc79e562a3b7542555d7a21ea/frontend/src/app/components/wp-table/configuration-modal/wp-table-configuration-relation-selector.ts#L14"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>